index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="An aspiring software developer with experience in desktop, web & mobile applications.">
+        <title>Portfolio</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+        <header>
+            <div id="time-utility">
+                <div>
+                    <img src="./icons/time-slot.png">
+                </div>
+                <div id="utilities">
+                    <div>
+                        <img src="./icons/cellular.png">
+                    </div>
+                    <div>
+                        <img src="./icons/Wifi.png">
+                    </div>
+                    <div>
+                        <img src="./icons/Battery.png">
+                    </div>
+                </div>
+            </div>
+            <div id="name-menu">
+                <h4>Md Tariq-ul Islam</h4>
+                <img src="./icons/Menu.svg">
+            </div>
+        </header>
+        <div id="welcome">          
+            <h1>Hey There.</br>I'm Tariq-ul Islam</h1>
+            <h2>I am a Software Developer</h2>
+            <p>I can help you build a product, feature or website look through some of my work and experience. If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            <ul id="social-media">
+                <li><img class="social-media-icon" src="./icons/github.svg" alt="Github"></li>
+                <li><img class="social-media-icon" src="./icons/Linkedin.svg" alt="Linked-in"></li>
+                <li><img class="social-media-icon" src="./icons/Angelist.svg" alt="Angelist"></li>
+                <li><img class="social-media-icon" src="./icons/twitter.svg" alt="Twitter"></li>
+                <li><img class="social-media-icon" src="./icons/medium.svg" alt="Medium"></li>
+            </ul>
```

Comment:
- [ ] Nice to use the `ul` and `li` tags to list the social media icons in your project. But you should use the `a` tag to wrap the `img` tag in your project. So that when you click on the social media icon it will take you to the social media page. :pray: